### PR TITLE
change max zoom level and basemap

### LINKED ISSUE #582: zoom level is not convenient
# Problem 

The maximum zoom is not granular enough. We need to be able to zoom at least to city level.

# Proposed Solution 

Increase the maximum level.

### PR DESCRIPTION
**Pull request by anitanh**
fix #582 

This PR:
- Increase maximum zoom level
- Change basemap, because the previous one got system unavailable (503)

Min zoom:
screenshot from 2018-07-24 12-17-42

Max zoom
screenshot from 2018-07-24 12-18-10

### CODE REVIEW

File: django_project/feti/static/js/scripts/views/map.js [hunk 1]
```diff
@@ -126,11 +126,11 @@ define([
             this.map.on('dblclick', function (e) {
             });
 
-            L.tileLayer('https://korona.geog.uni-heidelberg.de/tiles/roads/x={x}&y={y}&z={z}', {
+            L.tileLayer('https://{s}.tile.openstreetmap.fr/osmfr/{z}/{x}/{y}.png', {
```
Comment (meomancer) [inline django_project/feti/static/js/scripts/views/map.js:129]: hello @ann26 
is there any global openstreetmap? using com/org? (also update france version)
why we are using france domain?

**Review by meomancer — CHANGES_REQUESTED** [review]
please see my comment

File: django_project/feti/static/js/scripts/views/map.js (hunk 1 above)
Comment (anitanh) [reply django_project/feti/static/js/scripts/views/map.js:129]: hi @meomancer , yes there is, but the france domain has prettier basemap I think. Although now this basemap: `https://korona.geog.uni-heidelberg.de/tiles/roads/x={x}&y={y}&z={z}` is working, yesterday I got 503. Maybe I should change it back?

Comment (meomancer) [reply django_project/feti/static/js/scripts/views/map.js:129]: what do you think @cchristelis ?

**Comment by meomancer** [comment]
Maybe could you split the PR
one for zoom issue and another one is for basemap? @ann26 
(maybe close this one)

**Comment by anitanh** [comment]
sure @meomancer

**Pull request closed without merging**